core/lib/src/form/options.rs

```diff
@@ -1,6 +1,9 @@
 /// Form guard options.
 ///
 /// See [`Form#leniency`](crate::form::Form#leniency) for details.
+#[non_exhaustive] // TODO Remove when more than one option is available.
+                  // This attribute was added to avoid Clippy's "needless_update" warning inside
+                  // the function strict::init.
```

Comment:
This is fine, actually, no need for a `TODO`.

Reply:
Ok thanks, I'll remove the TODO.

I've added it because, since I didn't have context, I did not know if I had to remove `..opts` and rename the parameter `_opts` in this function :
```
fn init(opts: Options) -> Self::Context {
        T::init(Options { strict: true, ..opts })
    }
```
But the Clippy's page [needless_update](https://rust-lang.github.io/rust-clippy/master/index.html#needless_update) says It is ignored with `non_exhaustive` types because they are work in progress. Both solutions have implications at compile time. But I've felt like `Options` was work in progress in this case.

Somme warning like this one are a bit hard to fix without context. Moreover fixing hundreds of them in one PR is time consuming and can be very error prone. Which is why I've suggested in this PR's intro to ask people to fix their own warning.